documentation/related_projects/index.markdown

```diff
@@ -0,0 +1,40 @@
+---
+author: davetcoleman
+comments: false
+date: 2016-11-02 20:00:00+00:00
+layout: page
+slug: related_projects
+title: Related Projects
+---
+
+# Related Projects Aligned with MoveIt! Framework
+
+MoveIt! is designed to be easily extensible as separate plugins or projects. Below is a list of related projects that have been aligned with MoveIt! framework, in alphabetical order of project names.
+
+## bio_ik
+MoveIt kinematics_base plugin based on particle optimization & GA. For ease of use and compatibility with existing code, the bio-ik algorithm is encapsulated as a Moveit kinematics plugin. Therefore, bio-ik can be used as a direct replacement of the default Orocos/KDL-based IK solver. Given the name of an end-effector and a 6-DOF target pose, bio-ik will search a valid robot joint configuration that reaches the given target. [More Info](https://github.com/TAMS-Group/bio_ik)
+
+## descartes
+ROS-Industrial Special Project: Cartesian Path Planner. Descartes is a ROS-Industrial project for performing path-planning on under-defined Cartesian trajectories. Descartes uses trajectory points, robot models, and planners to generate an joint-trajectory that complies with the constraints of a given process. [More Info](https://github.com/ros-industrial-consortium/descartes)
+
+## industrial_moveit
+This stack is part of the ROS Industrial program. It currently contains packages that are meant for use with the MoveIt packages. [More Info](https://github.com/ros-industrial/industrial_moveit)
+
+## moveit_grasps
+Geometric grasping generator library for cuboids https://picknik.ai/. A basic grasp generator for objects such as blocks or cylinders for use with the MoveIt! pick and place pipeline. Does not consider friction cones or other dynamics.
+
+Its current implementation takes as input a pose vector (postition and orientation) and generates a large number of potential grasp approaches and directions. Also includes a grasp filter for removing kinematically infeasible grasps via threaded IK solvers. [More Info](	https://github.com/PickNikRobotics/moveit_grasps)
+
+## moveit_task_constructor
```

Comment:
the moveit_task_constructor and moveit_visual_tools are part of MoveIt! and live in ros-planning, so should not be listed here but rather http://moveit.ros.org/documentation/source-code-api/